io_burst/burst_io.sh

```diff
@@ -330,6 +330,13 @@ while [[ $# -gt 0 ]]; do
 		;;
         esac
 done
+
+# Install needed packages based on what's listed in the wrapper's json file
+${TOOLS_BIN}/package_tool --wrapper_config ${run_dir}/io_burst-wrapper.json
```

Comment:
Need to add `--no_packages $to_no_pkg_install` to allow for no-package install runs